src/Symfony/Component/Validator/Constraints/Expression.php

```diff
@@ -54,12 +54,4 @@ public function getTargets()
     {
         return array(self::CLASS_CONSTRAINT, self::PROPERTY_CONSTRAINT);
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function validatedBy()
-    {
-        return 'validator.expression';
```

Comment:
This is a BC break. Before this change I was able to replace the validator for the `Expression` constraint with my own service. That's now not possible anymore but I will have to create my own validator factory instead.